scenes/game_elements/characters/npcs/npc_prop/fray_hammerman.tscn

```diff
@@ -129,6 +129,7 @@ libraries = {
 &"": SubResource("AnimationLibrary_003jk")
 }
 autoplay = "default"
+speed_scale = 0.9
```

Comment:
Is changing the speed of the hammer men from 1 to 0.9 intentional? Is it even noticeable?

Reply:
Oh, while testing I noticed that the 2 hammer men now go at different speed, nice.

Reply:
We can do that after. Merging for now.